.github/workflows/main.yml

```
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ main ]
```

Why only branch `main`? The homework task says:
>   - update [status checks](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-status-checks) on all PRs to your repository

---

You can hook this just to push or pull requests, independent from the branch.
```suggestion
    on: [push, pull_request]
```